src/components/30-organisms/table-sortable/index.js

```diff
@@ -124,11 +136,12 @@ class AXATableSortable extends LitElement {
   }
 
   sortByIndex(index, actualSortAs) {
-    const sortAs = actualSortAs === ASC ? DESC : ASC;
+    const sortAs = actualSortAs === ASC ? DESC : ASC; // TODO why? in my opinion aria is set to early
```

Comment:
@NoahLiechti what do you mean here?

Reply:
ascending and descending sort is in my opinion reversed

Reply:
ascending is 0 - 20, a - z and 01.01.2020 - 04.03.2020 right?

Reply:
ahh you are right:)